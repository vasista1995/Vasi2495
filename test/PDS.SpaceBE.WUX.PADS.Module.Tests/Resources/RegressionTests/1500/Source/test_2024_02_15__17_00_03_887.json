{
  "IdSource": "SPACEACT2:WUX:BE:VE335969C03:ACWUS:D_WireBond_Mould:StitchPull:33241:200697227:0",
  "Data_flat_Metadata": {
    "Package": "PG-SOT23-3-5",
    "ParameterClass": "NON_PPAP",
    "Segment": "DS",
    "Module": "M1_ASSY",
    "FourDReport": "*",
    "ChipType": "L8062B65221",
    "SalesName": "BSS308PE H6327",
    "BusinessSegment": "15",
    "WireSize": "Cu wire 43um",
    "SolderGlue": "null",
    "DieSize": "0.8",
    "Operator": "30075946",
    "CartridgeID": "DT0001",
    "Grade": ".",
    "Baunumber": "70012864",
    "GaugeID": "0.8",
    "Line": "SOT23-8UP",
    "Shift": "C",
    "Process": "WireBond",
    "Lot": "VE335969C03",
    "ParameterName": "StitchPull",
    "ParameterUnit": "g",
    "ParameterOper": "D_WireBond_Mould",
    "ParameterFacility": "ACWUS",
    "SiteKey": "WUX",
    "ChannelId": "33241",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
    "CreatedTimestamp": "2023-10-06T15:05:48Z",
    "UpdatedTimestamp": "2023-10-06T15:05:48Z",
    "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
    "ExportedTimestampUtc": "2023-10-06T15:05:48Z",
    "ExportedTimestamp": "2023-10-06T15:05:48Z",
    "SampleTimestampUtc": "2023-10-06T07:05:48Z",
    "SampleTimestamp": "2023-10-06T15:05:48Z",
    "SourceDataLevel": "C",
    "Equipment": "DS-S521",
    "ChannelName": "Stitch_PullForce/SOT23-8UP/Cu/43um",
    "ChannelDescr": "Created from LDS \nexclude list by \naction 'Create SPC Channel'",
    "RvStoreFlag": "Y",
    "CalcCreated": "04/13/2023 14:40:00",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 4307369,
    "ViolCount": 0,
    "CalculationStrategy": "ND 4.5 Sigma",
    "LdsID": "1500",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 9.0,
        "SpecTarget": 28.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYYNYYYYYYYYYYYYYYYYYYYYYYNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "32,850625",
        "Median": "33,2105",
        "Sigma": "1,96228619854495",
        "Min": "29,698",
        "Max": "35,341",
        "Q1": "31,65025",
        "Q3": "34,29"
      },
      "ControlLimits": {
        "ctrl_mean_low": 21.082331423304,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 28.3275078596038,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 35.5726842959035,
        "CtrlMeanHighEnabled": "Y",
        "ctrl_low": 17.5080622691884,
        "CtrlLowEnabled": "Y",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "Y",
        "ctrl_target": 28.3272942884802,
        "ctrl_high": 39.1465263077719,
        "CtrlHighEnabled": "Y",
        "CtrlSigmaLowEnabled": "N",
        "ctrl_sigma_target": 1.76698688822979,
        "ctrl_sigma_high": 5.42706800609763,
        "CtrlSigmaHighEnabled": "Y",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 5.16094649975764,
        "ctrl_range_high": 18.0766348787636,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 31.59565625,
        "ExtEwmaMV": 31.8560035124211,
        "ExtMsMV": 1.96345875465975,
        "EwmaS": 2.1565080529345,
        "EwmaR": 6.09721080552962,
        "ExtEwmaMVLCL": 25.9075325533123,
        "ExtEwmaMVCenter": 28.3272942884802,
        "ExtEwmaMVUCL": 30.7470560236481,
        "ExtMaMVLCL": 26.6162643566872,
        "ExtMaMVCenter": 28.3272942884802,
        "ExtMaMVUCL": 30.0383242202732,
        "EwmaSLCL": 0.94311042539142,
        "EwmaSCenter": 1.82189420626948,
        "EwmaSUCL": 2.70067798714754,
        "EwmaRLCL": 2.69539460759517,
        "EwmaRCenter": 5.34746128318584,
        "EwmaRUCL": 7.99952795877651,
        "ExtMSMVLCL": 0.00217780837561695,
        "ExtMSMVCenter": 1.02698904506354,
        "ExtMSMVUCL": 4.12545476592162,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 17.5080622691884,
        "RawUal": 39.1465263077719,
        "SigmaLal": -1.58164858632525,
        "SigmaUal": 9.08714912396548,
        "RangeLal": -5.3610778804741,
        "RangeUal": 30.9923232577696
      }
    },
    "Data_1list_RawValues": [
      {
        "Wafer": "-",
        "SampleId": 200697227,
        "Seqnr": 7,
        "Value": 32.635,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "Wafer": "-",
        "SampleId": 200697227,
        "Seqnr": 1,
        "Value": 34.239,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "Wafer": "-",
        "SampleId": 200697227,
        "Seqnr": 8,
        "Value": 34.443,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "Wafer": "-",
        "SampleId": 200697227,
        "Seqnr": 2,
        "Value": 30.694,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "Wafer": "-",
        "SampleId": 200697227,
        "Seqnr": 3,
        "Value": 31.969,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "Wafer": "-",
        "SampleId": 200697227,
        "Seqnr": 4,
        "Value": 35.341,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "Wafer": "-",
        "SampleId": 200697227,
        "Seqnr": 5,
        "Value": 33.786,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "Wafer": "-",
        "SampleId": 200697227,
        "Seqnr": 6,
        "Value": 29.698,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 8
  },
  "ProductionAction": {
    "Id": "SPACEACT1:WUX:BE:ACWUS:D_WireBond_Mould:StitchPull:33241",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:VE335969C03",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "WUX:BE:VE335969C03:ACWUS:D_WireBond_Mould:StitchPull:33241:200697227:0"
  }
}