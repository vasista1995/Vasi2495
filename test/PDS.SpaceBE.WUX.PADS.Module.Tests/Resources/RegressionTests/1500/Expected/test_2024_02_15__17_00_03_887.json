{
  "Data_flat_Metadata": {
    "Baunumber": "70012864",
    "GaugeID": "0.8",
    "Line": "SOT23-8UP",
    "Shift": "C",
    "Process": "WireBond",
    "MeasLot": "VE335969C03",
    "ParameterOper": "D_WireBond_Mould",
    "ParameterFacility": "ACWUS",
    "SiteKey": "WUX",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-10-06T07:05:48Z",
    "BeginTimestamp": "2023-10-06T15:05:48Z",
    "EndTimestampUtc": "2023-10-06T07:05:48Z",
    "EndTimestamp": "2023-10-06T15:05:48Z",
    "Equipment": "DS-S521"
  },
  "Data_flat_StrangeMetadata": {
    "Package": "PG-SOT23-3-5",
    "SalesName": "BSS308PE H6327",
    "BusinessSegment": "15",
    "Wiresize": "Cu wire 43um",
    "SolderGlue": "null",
    "DieSize": "0.8",
    "ChipType": "L8062B65221",
    "Operator": "30075946",
    "CartridgeID": "DT0001",
    "Grade": "."
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Wafer": "-",
          "Seqnr": 7,
          "SampleId": 200697227,
          "Value": 32.635,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:05:48Z",
          "SampleTimestampUtc": "2023-10-06T07:05:48Z",
          "CreatedTimestamp": "2023-10-06T15:05:48Z",
          "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
          "UpdatedTimestamp": "2023-10-06T15:05:48Z",
          "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
          "NumViolations": 0,
          "SampleMean": 32.850625,
          "SampleMin": 29.698,
          "SampleMax": 35.341,
          "SampleMedian": 33.2105,
          "SampleStdev": 1.96228619854495,
          "SampleQ1": 31.65025,
          "SampleQ3": 34.29,
          "SampleSize": 8
        },
        {
          "Wafer": "-",
          "Seqnr": 1,
          "SampleId": 200697227,
          "Value": 34.239,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:05:48Z",
          "SampleTimestampUtc": "2023-10-06T07:05:48Z",
          "CreatedTimestamp": "2023-10-06T15:05:48Z",
          "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
          "UpdatedTimestamp": "2023-10-06T15:05:48Z",
          "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
          "NumViolations": 0,
          "SampleMean": 32.850625,
          "SampleMin": 29.698,
          "SampleMax": 35.341,
          "SampleMedian": 33.2105,
          "SampleStdev": 1.96228619854495,
          "SampleQ1": 31.65025,
          "SampleQ3": 34.29,
          "SampleSize": 8
        },
        {
          "Wafer": "-",
          "Seqnr": 8,
          "SampleId": 200697227,
          "Value": 34.443,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:05:48Z",
          "SampleTimestampUtc": "2023-10-06T07:05:48Z",
          "CreatedTimestamp": "2023-10-06T15:05:48Z",
          "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
          "UpdatedTimestamp": "2023-10-06T15:05:48Z",
          "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
          "NumViolations": 0,
          "SampleMean": 32.850625,
          "SampleMin": 29.698,
          "SampleMax": 35.341,
          "SampleMedian": 33.2105,
          "SampleStdev": 1.96228619854495,
          "SampleQ1": 31.65025,
          "SampleQ3": 34.29,
          "SampleSize": 8
        },
        {
          "Wafer": "-",
          "Seqnr": 2,
          "SampleId": 200697227,
          "Value": 30.694,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:05:48Z",
          "SampleTimestampUtc": "2023-10-06T07:05:48Z",
          "CreatedTimestamp": "2023-10-06T15:05:48Z",
          "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
          "UpdatedTimestamp": "2023-10-06T15:05:48Z",
          "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
          "NumViolations": 0,
          "SampleMean": 32.850625,
          "SampleMin": 29.698,
          "SampleMax": 35.341,
          "SampleMedian": 33.2105,
          "SampleStdev": 1.96228619854495,
          "SampleQ1": 31.65025,
          "SampleQ3": 34.29,
          "SampleSize": 8
        },
        {
          "Wafer": "-",
          "Seqnr": 3,
          "SampleId": 200697227,
          "Value": 31.969,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:05:48Z",
          "SampleTimestampUtc": "2023-10-06T07:05:48Z",
          "CreatedTimestamp": "2023-10-06T15:05:48Z",
          "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
          "UpdatedTimestamp": "2023-10-06T15:05:48Z",
          "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
          "NumViolations": 0,
          "SampleMean": 32.850625,
          "SampleMin": 29.698,
          "SampleMax": 35.341,
          "SampleMedian": 33.2105,
          "SampleStdev": 1.96228619854495,
          "SampleQ1": 31.65025,
          "SampleQ3": 34.29,
          "SampleSize": 8
        },
        {
          "Wafer": "-",
          "Seqnr": 4,
          "SampleId": 200697227,
          "Value": 35.341,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:05:48Z",
          "SampleTimestampUtc": "2023-10-06T07:05:48Z",
          "CreatedTimestamp": "2023-10-06T15:05:48Z",
          "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
          "UpdatedTimestamp": "2023-10-06T15:05:48Z",
          "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
          "NumViolations": 0,
          "SampleMean": 32.850625,
          "SampleMin": 29.698,
          "SampleMax": 35.341,
          "SampleMedian": 33.2105,
          "SampleStdev": 1.96228619854495,
          "SampleQ1": 31.65025,
          "SampleQ3": 34.29,
          "SampleSize": 8
        },
        {
          "Wafer": "-",
          "Seqnr": 5,
          "SampleId": 200697227,
          "Value": 33.786,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:05:48Z",
          "SampleTimestampUtc": "2023-10-06T07:05:48Z",
          "CreatedTimestamp": "2023-10-06T15:05:48Z",
          "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
          "UpdatedTimestamp": "2023-10-06T15:05:48Z",
          "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
          "NumViolations": 0,
          "SampleMean": 32.850625,
          "SampleMin": 29.698,
          "SampleMax": 35.341,
          "SampleMedian": 33.2105,
          "SampleStdev": 1.96228619854495,
          "SampleQ1": 31.65025,
          "SampleQ3": 34.29,
          "SampleSize": 8
        },
        {
          "Wafer": "-",
          "Seqnr": 6,
          "SampleId": 200697227,
          "Value": 29.698,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-06T15:05:48Z",
          "SampleTimestampUtc": "2023-10-06T07:05:48Z",
          "CreatedTimestamp": "2023-10-06T15:05:48Z",
          "CreatedTimestampUtc": "2023-10-06T07:05:48Z",
          "UpdatedTimestamp": "2023-10-06T15:05:48Z",
          "UpdatedTimestampUtc": "2023-10-06T07:05:48Z",
          "NumViolations": 0,
          "SampleMean": 32.850625,
          "SampleMin": 29.698,
          "SampleMax": 35.341,
          "SampleMedian": 33.2105,
          "SampleStdev": 1.96228619854495,
          "SampleQ1": 31.65025,
          "SampleQ3": 34.29,
          "SampleSize": 8
        }
      ],
      "ParameterClass": "NON_PPAP",
      "Segment": "DS",
      "Module": "M1_ASSY",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "N",
          "SpecLowEnabled": "Y",
          "spec_low": 9.0,
          "target": 28.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 21.082331423304,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 28.3275078596038,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 35.5726842959035,
          "CtrlMeanHighEnabled": "Y",
          "ctrl_low": 17.5080622691884,
          "CtrlLowEnabled": "Y",
          "ctrl_target": 28.3272942884802,
          "ctrl_high": 39.1465263077719,
          "CtrlHighEnabled": "Y",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "Y",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "Y",
          "ctrl_stdev_target": 1.76698688822979,
          "ctrl_stdev_high": 5.42706800609763,
          "CtrlStdevHighEnabled": "Y",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 5.16094649975764,
          "ctrl_range_high": 18.0766348787636,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 8,
        "base_count": 8,
        "flagged_count": 0,
        "mean": 32.850625,
        "median": 33.210499999999996,
        "stdev": 1.8355506624375693,
        "range": 5.643000000000001,
        "min": 29.698,
        "max": 35.341,
        "q02": 29.83744,
        "q05": 30.046599999999998,
        "q25": 31.650250000000003,
        "q75": 34.28999999999999,
        "q95": 35.0267,
        "q98": 35.21528,
        "NumViolations": 0,
        "Samples": "200697227"
      },
      "ParameterName": "StitchPull",
      "ParameterUnit": "g",
      "ChannelId": "33241",
      "SourceDataLevel": "C",
      "channel_name": "Stitch_PullForce SOT23-8UP Cu 43um",
      "channel_description": "Created from LDS  exclude list by  action 'Create SPC Channel'",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-10-06T07:05:48Z",
      "created_timestamp": "2023-10-06T15:05:48Z",
      "updated_timestamp_utc": "2023-10-06T07:05:48Z",
      "updated_timestamp": "2023-10-06T15:05:48Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-10-06T07:05:48Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:WUX:BE:ACWUS:D_WireBond_Mould:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:VE335969C03",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "WUX:BE:VE335969C03:ACWUS:D_WireBond_Mould:StitchPull:33241:200697227:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-10-06T15:05:48Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot:VE335969C03:SPACEAGGED2:WUX:BE:ACWUS:D_WireBond_Mould::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "WUX"
  }
}